Validation/Tools/examples/printConversionInfo.py

```diff
@@ -6,7 +6,7 @@
 from DataFormats.FWLite import Events, Handle
 
 from FWCore.ParameterSet.VarParsing import VarParsing
-options = VarParsing ('python')
+options = VarParsing ('python3')
```

Comment:
Given the behavior of `VarParsing` constructor
https://github.com/cms-sw/cmssw/blob/49cd0eb9a47e85d38023c5c158e60be18687ee04/FWCore/ParameterSet/python/VarParsing.py#L38-L41
this change should be reverted.

On the other hand, I have no idea if this example is still useful or actually works (it has not been changed except for technical migrations since its introduction in 2009).

Reply:
Reverted

Reply:
Thanks (and sorry for noticing this late)